ufc/flags-v1.json

```diff
@@ -0,0 +1,208 @@
+      },
+      "allocations": [
+        {
+          "key": "internal users",
+          "rules": [
+            {
+              "conditions": [
+                {
+                  "attribute": "email",
+                  "operator": "MATCHES",
+                  "value": ".*@mycompany.com"
+                }
+              ]
+            }
+          ],
+          "startAt": null,
+          "endAt": null,
+          "splits": [
+            {
+              "variationKey": "green",
+              "shards": []
+            }
+          ],
+          "doLog": false
+        },
+        {
+          "key": "experiment",
+          "rules": [],
+          "startAt": null,
+          "endAt": null,
+          "splits": [
+            {
+              "variationKey": "control",
+              "shards": [
+                {
+                  "salt": "traffic-new-user-onboarding-experiment",
+                  "ranges": [{ "start": 0, "end": 2000 }]
+                },
+                {
+                  "salt": "split-new-user-onboarding-experiment",
+                  "ranges": [{ "start": 0, "end": 5000 }]
+                }
+              ]
+            },
+            {
+              "variationKey": "red",
+              "shards": [
+                {
+                  "salt": "traffic-new-user-onboarding-experiment",
+                  "ranges": [{ "start": 0, "end": 2000 }]
+                },
+                {
+                  "salt": "split-new-user-onboarding-experiment",
+                  "ranges": [{ "start": 5000, "end": 8000 }]
+                }
+              ]
+            },
+            {
+              "variationKey": "green",
+              "shards": [
+                {
+                  "salt": "traffic-new-user-onboarding-experiment",
+                  "ranges": [{ "start": 0, "end": 2000 }]
+                },
+                {
+                  "salt": "split-new-user-onboarding-experiment",
+                  "ranges": [{ "start": 8000, "end": 10000 }]
+                }
+              ]
+            }
+          ],
+          "doLog": true
+        },
+        {
+          "key": "rollout",
+          "rules": [
+            {
+              "conditions": [
+                {
+                  "attribute": "country",
+                  "operator": "ONE_OF",
+                  "value": ["US", "Canada", "Mexico"]
+                }
+              ]
+            }
+          ],
+          "startAt": null,
+          "endAt": null,
+          "splits": [
+            {
+              "variationKey": "blue",
+              "shards": [
+                {
+                  "salt": "split-new-user-onboarding-rollout",
+                  "ranges": [{ "start": 0, "end": 8000 }]
+                }
+              ],
+              "extraLogging": {
+                "allocationvalue_type": "rollout",
+                "owner": "Alice"
+              }
+            }
+          ],
+          "doLog": true
+        }
```

Comment:
I'm confused by this allocation.
1. Is it intended to be below the experiment? I could understand this to mean "we rolled out blue as the new control group to NA, and now we're running an experiment" ... but as the experiment traffic increases, more and more NA subjects will actually end up getting the _previous_ `control`?
2. Why is the end 8000? In what sense is this a rollout if it lets 20% of traffic pass through?
3. Should we have a default allocation after it? Everyone that passes through is (implicitly) getting `null` at the moment. We should enforce a default allocation at UFC generation time. (If you want to make sure the SDK is robust to that not working as intended, it should be a dedicated test.)

Reply:
The goal here is to test all functionality of the UFC, so the config might not make the most sense in practice.